### ISSUE TITLE
logits and labels must be broadcastable

### ISSUE THREAD

**Issue by wangxianzhe** [leimao/DeepLab_v3#11]
Using other data sets appears error
' logits and labels must be broadcastable: logits_size=[627500,21] labels_size=[628000,21]'
